.vscode/tasks.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Configure Waf for host OS: x86",
+            "type": "shell",
+            "problemMatcher": [],
+            "command": "./waf configure -T release --prefix=out/",
```

Comment:
On macOS you need to call `python3 waf`

Reply:
The entire configuration is made on the assumption that macOS is still not supported, [as per the Wiki](https://github.com/nillerusr/source-engine/wiki/How-to-build#on-macos).
Is the wiki wrong? Is macOS now supported?

Reply:
macOS is supported. wiki is not updated yet

Reply:
On that note, is newer Android NDK supported?

Reply:
I would still like to know if Windows is supported, although I have gone ahead and assumed it is.